### ISSUE TITLE
Problem with pagination

### ISSUE THREAD

**Issue by supadrupa** [barseghyanartur/django-elasticsearch-dsl-drf#69]
If elasticsearch empty return KeyError 'suggest'

the problem is here
[https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/pagination.py#L130](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/pagination.py#L130)
and 
[https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/pagination.py#L236](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/pagination.py#L236)

maybe  
```python
return queryset.execute().to_dict()['suggest']
```
update code
```python
return queryset.execute().to_dict().get('suggest')
```

**Comment by barseghyanartur** [comment]
Thanks. Could you check the tests? It weird it wasn't cought. It would be awesome if you could come up with pull request.

**Comment by barseghyanartur** [comment]
@supadrupa:

Solved in 0.13.1.
